src/api/GoalsAPI/index.ts

```diff
@@ -0,0 +1,46 @@
+import { db, GoalItem } from '@models';
+import { getJustDate } from '@src/utils';
+
+export const resetDatabase = () => db.transaction('rw', db.goalItems, async () => {
+  await Promise.all(db.tables.map((table) => table.clear()));
+});
+
+export const addGoal = (goalDetails :{
+  title: string,
+  duration: string,
+  repeat: string,
+  start: Date,
+  finish: Date,
+  at: Date}) => {
```

Comment:
What is this `at`? Were you able to find a createdAt field for database items in DexieJS?

Reply:
No, I will create an issue on this.
I made an alternate for now. **createdAt** will act as a timestamp. 
**at** is removed

Reply:
@Tushar-4781 there is still an `at` here at line 14. Please change that to `createdAt too`

Reply:
I made an alternate for now. **createdAt** will act as a timestamp.

Reply:
> @Tushar-4781 there is still an `at` here at line 14. Please change that to `createdAt too`

Kindly check the second commit. It is dedicated to that